python/aigverse/benchmarks/_cache.py

```
@@ -0,0 +1,79 @@
+
+    Applies process-wide and is intended to be called once during setup; it is
+    not thread-safe.
+
+    Args:
+        path: The directory to use, or ``None`` to clear a previous override and
+            fall back to the environment variable and the platform default.
+
+    Returns:
+        The resolved directory, or ``None`` if the override was cleared.
+    """
+    global _override  # ruff: ignore[global-statement]
```

Ruff (0.16.0)

[warning] Invalid rule code in suppression: global-statement

Enable `lint.preview` to use rule names

(RUF102)